Networking/NetworkingTests/Requests/DotcomRequestTests.swift

```diff
@@ -91,6 +95,78 @@ class DotcomRequestTests: XCTestCase {
             XCTAssertEqual(value, sampleParameters[key])
         }
     }
+
+    // MARK: `RESTRequest` conversion
+
+    func test_it_is_converted_into_RESTRequest_for_wpMark2_when_availableAsRESTRequest_is_true() throws {
+        // Given
+        let request = try DotcomRequest(wordpressApiVersion: .wpMark2,
+                                        method: .post,
+                                        path: sampleRPC,
+                                        parameters: sampleParameters,
+                                        availableAsRESTRequest: true)
+
+        // When
+        let output = try XCTUnwrap(request.asRESTRequest(with: sampleSiteAddress))
+
+        // Then
+        XCTAssertEqual(output.apiVersionPath, WooAPIVersion.none.path)
+        XCTAssertEqual(output.method, .post)
+        XCTAssertEqual(output.path, WordPressAPIVersion.wpMark2.path + sampleRPC)
+        let params = try XCTUnwrap(output.parameters as? [String: String])
+        XCTAssertEqual(params, sampleParameters)
+        XCTAssertEqual(output.siteURL, sampleSiteAddress)
+    }
+
+    func test_initializing_RESTRequest_throws_for_non_WPOrg_endpoints__even_if_availableAsRESTRequest_is_true() throws {
```

Comment:
super nit:
```suggestion
    func test_initializing_RESTRequest_throws_for_non_WPOrg_endpoints_even_if_availableAsRESTRequest_is_true() throws {
```

Reply:
Done in 822409a8acff04fbf4f3afaa5cb204427402b5fa